packages/TesterApp/webpack.config.js

```diff
@@ -1,7 +1,7 @@
 const path = require('path');
-const webpack = require('webpack');
 const TerserPlugin = require('terser-webpack-plugin');
 const ReactNative = require('@callstack/repack');
+const webpack = require('../../node_modules/webpack');
```

Comment:
Can you add it as a comment please? It's a workaround, so it's worth an annotation

Reply:
This is only temporary, because with upcoming improvements to `webpack` handling in Repack, it will no longer be necessary to import `webpack` inside the config anymore.